src/index.js

```diff
@@ -412,7 +413,7 @@ export class Select extends Component {
     const tab = event.key === 'Tab' && !event.shiftKey;
     const shiftTab = event.shiftKey && event.key === 'Tab';
 
-    if (arrowDown && !state.dropdown) {
+    if ((arrowDown || enter) && !state.dropdown) {
```

Comment:
How will enter do if component is inside form?

Reply:
Hmmm sorry, I'm not sure. You mean like a form the user needs to submit? Would we need to add some kind of `preventDefault` or something to prevent unwanted form submission?

Reply:
I have to say enter is a bit problematic here from all perspectives. 
I looked at a few other examples and also tried to play with select myself using keyboard, I honestly think arrow down is the correct way to go. I have real concerns about allowing hitting enter key here.

Reply:
ok @sanusart, thanks for investigating it. We haven't received complaints about our current UX for this, so afaik it's not a high priority for us.